proto/cometbft/privval/v1/types.proto

```diff
@@ -20,8 +19,11 @@ message PubKeyRequest {
 
 // PubKeyResponse is a response message containing the public key.
 message PubKeyResponse {
-  cometbft.crypto.v1.PublicKey pub_key = 1 [(gogoproto.nullable) = false];
```

Comment:
We need to discuss how we'll be managing the proto versions from now on... the `v1` proto versions match CometBFT `v1.0.0`, but now these changes (and other we're doing on `main`) should trigger a bump in the proto version, I think.
TBH, I haven't given it a deep thought... it would probably be worth discussing in our team meeting.
This shouldn't be blocking this PR, BTW.

Reply:
Since we're backporting this to v1, no version bump is needed. That was my thought, at least.

Reply:
Oh, of course... spot on!